.github/workflows/ci-core-reusable.yml

```diff
@@ -203,10 +203,10 @@ jobs:
       - name: Show logs
         if: always()
         run: |
-          ci_run cat server.log
-          ci_run cat contract_verifier.log
-          ci_run cat core/tests/revert-test/revert.log
-          ci_run cat core/tests/upgrade-test/upgrade.log
+          ci_run cat server.log || true
+          ci_run cat contract_verifier.log || true
+          ci_run cat core/tests/revert-test/revert.log || true
+          ci_run cat core/tests/upgrade-test/upgrade.log || true
```

Comment:
Can you please change this to `cat` a single file within the single `- name` directive while you are at it? It would improve things so much more if all logs were not squashed together.

Reply:
great idea! I added this change